.github/workflows/release.yml

```diff
@@ -0,0 +1,62 @@
+name: Release
+on:
+  workflow_dispatch:
+    inputs:
+      package:
+        description: 'The published name of a single package to release'
+        type: choice
+        required: false
+        options:
+          - discord.js
+          - '@discordjs/brokers'
+          - '@discordjs/builders'
+          - '@discordjs/collection'
+          - '@discordjs/core'
+          - 'create-discord-bot'
+          - '@discordjs/docgen'
+          - '@discordjs/formatters'
+          - '@discordjs/next'
+          - '@discordjs/proxy'
+          - '@discordjs/rest'
+          - '@discordjs/util'
+          - '@discordjs/voice'
+          - '@discordjs/ws'
+      exclude:
+        description: 'Comma separated list of packages to exclude from release (if not depended upon)'
+        required: false
+        type: string
+        default: '@discordjs/docgen,@discordjs/next'
+jobs:
+  npm-publish:
+    name: npm publish
```

Comment:
Would probably just say "publish" as we use pnpm and not npm.

Reply:
it's publishing to npm